### ISSUE TITLE
Flutter Module not found in Xcode

### ISSUE THREAD

**Issue by CJCodebuild** [atiqsamtia/WordPress-App-with-Flutter#14]
Flutter Module Not found in Xcode 
<img width="757" alt="Screen Shot 2022-10-12 at 9 50 01 AM" src="https://user-images.githubusercontent.com/115641985/195360691-a6773336-3e4a-4184-91c2-b906fd50ec72.png">

Also framworks were added via adding podfiles but cant add pofiles to get the framworks in there. dont no what to do any ideas?

<img width="588" alt="Screen Shot 2022-10-12 at 9 51 54 AM" src="https://user-images.githubusercontent.com/115641985/195361183-508a3efc-cc69-4880-a308-39145842e04a.png">

shouldn't it be a flutter framework? Please help with this I am using Xcode and cant get this to launch on the simulator. I used flutter doctor everything was updated all green check marks, for some reason it wont allow me to 'flutter packages get' your Git and then run it. don't no what to do i went threw so much trouble shooting already you have no idea.

**Comment by atiqsamtia** [comment]
This project was generated years ago, we did an update for Android Embedding but did not test on Xcode, I'll check in morning on Xcode maybe it'll work fine if we delete ios dir and then run `flutter create . ` but I'll test on Mac and push the changes.

**Comment by CJCodebuild** [comment]
ios dir yiou mean should be in the folder? so once we delete that run flutter create on the terminal and then run it on xcode to seeif it works?

**Comment by CJCodebuild** [comment]
thanks for your help

**Comment by atiqsamtia** [comment]
Correct, delete `ios` dir and run `flutter create . ` in the terminal for the main project directory. and then run the project via code.  Do put your findings here as well if it works or not. 

I'm on Windows right now and can't test today.

**Comment by CJCodebuild** [comment]
im going to give it a try and tell you now if it works

**Comment by CJCodebuild** [comment]
here is what im getting after putting the rule into the terminal zsh: command not found: flutter

**Comment by CJCodebuild** [comment]
i put the ios folder in the trash temporarily and used to terminal, cd to the directory where i had ios and did flutter create still no luck so i brought the old file back in its place. This is bazzar and not only that the fact that it wont normally work on xcode is bazzare as well

**Comment by CJCodebuild** [comment]
once i finish up with xcode i will be also testing the android files on there to see if it works as an android app. but for now i want to focus on getting this to work on xcode first and foremost thanks

**Comment by atiqsamtia** [comment]
I pushed a new branch. `ios_upgrade` do test with it

**Comment by CJCodebuild** [comment]
ok i will check it out now

**Comment by CJCodebuild** [comment]
where is the new branch located ?

**Comment by atiqsamtia** [comment]
https://github.com/atiqsamtia/WordPress-App-with-Flutter/tree/ios_upgrade

You can also run `git pull` and then `git checkout ios_upgrade`

**Comment by CJCodebuild** [comment]
perfect and just so you know im running a brand new Mac M1 24" ok I will try it i new that the verisons for minimum versions ios had to be updated i did already start that okay let me try all the rest you did and ill tell you now if that works

**Comment by CJCodebuild** [comment]
Same issues again look = No such module 'Flutter'

**Comment by CJCodebuild** [comment]
i even added a Podfile and its not working but when i added flutter to it it wont allow me to for some reason i don't understand its somehow not letting the framework to allow for the activation of the module to work at all

**Comment by CJCodebuild** [comment]
# platform :ios, '9.0'

target 'my_project' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!

  # Pods for my_project
flutter_application_path = '../flutter_project'
  load File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')
  install_all_flutter_pods(flutter_application_path)

target 'my_projectTests' do
    inherit! :search_paths
    # Pods for testing
    install_all_flutter_pods(flutter_application_path)
  end

  target 'my_projectUITests' do
    # Pods for testing
    install_all_flutter_pods(flutter_application_path)
  end

end

**Comment by CJCodebuild** [comment]
when i go pod install in the dir it says [!] Invalid `Podfile` file: cannot load such file -- ../Podfile/.ios/Flutter/podhelper.rb.

**Comment by CJCodebuild** [comment]
this just isnt working im out of ideas

**Comment by CJCodebuild** [comment]
Hi their, Just wanted to know if you tried it on Xcode yet and had any luck please let me know thanks

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
